Tests/resources/fixtures/TestController.php

```diff
@@ -11,11 +11,15 @@ public function init()
 
     public function productAction()
     {
-        $useStaticLoader = PHP_VERSION_ID >= 50600 && !defined('HHVM_VERSION') && (!function_exists('zend_loader_file_encoded') || !zend_loader_file_encoded());
-
         $this->trans('Fingers', [], 'admin.product.help');
     }
 
+    public function nothingAction()
+    {
+        // the parser should not capture this
+        $version = phpversion();
```

Comment:
No need for a call of `$this->trans` here to validate the test?

Reply:
I dont see the need 🤔 without the related code fix, this will make the parser crash. The expected behavior is "when parsing a file that contains function without arguments, the parser ignores them".

Reply:
Alright, I just wanted to be sure it fits the need for this test ;)